tooling/github/setup/action.yml

```diff
@@ -7,7 +7,7 @@ runs:
     - uses: pnpm/action-setup@v4
     - uses: actions/setup-node@v4
       with:
-        node-version: 20
+        node-version: 22
```

Comment:
Why 22? If this is the new required version, please update it everywhere.
Also, would be nice to have a seperate commit (and PR) for this

Reply:
It did not work with 20 (because all the others are already updated to node 22, see #1385)
I'll create a quick seperate PR for it

Reply:
PR created: https://github.com/homarr-labs/homarr/pull/1429